apps/gk_species_radiation.c

```diff
@@ -91,6 +99,31 @@ gk_species_radiation_init(struct gkyl_gyrokinetic_app *app, struct gk_species *s
   rad->nvnu = mkarr(app->use_gpu, app->basis.num_basis, s->local_ext.volume);
   rad->nvsqnu_surf = mkarr(app->use_gpu, surf_mu_basis.num_basis, s->local_ext.volume);
   rad->nvsqnu = mkarr(app->use_gpu, app->basis.num_basis, s->local_ext.volume);
+  rad->vtsq = mkarr(app->use_gpu, app->confBasis.num_basis, app->local_ext.volume);
+
+  // Needed arrays for calculating temperature
+  rad->boundary_corrections = mkarr(app->use_gpu, 2*app->confBasis.num_basis, app->local_ext.volume);
+  rad->prim_moms = mkarr(app->use_gpu, 2*app->confBasis.num_basis, app->local_ext.volume);
+
+  rad->vtsq_min = 1.0;
```

Comment:
@akashukla @jRoeltgen please
- place this higher up so it doesn't get lost deep in the code, given that this is an important hardcoded parameter
- Are you sure you wanna set it to 1? 1 what? does it have the correct units?
- Please add a comment of what this is, what it's for, and why you picked this value.